libs/barista-components/filter-field/src/filter-field.ts

```diff
@@ -1119,6 +1119,10 @@ export class DtFilterField<T = any>
               recentRangeValue.operator as DtFilterFieldRangeOperator,
             );
           }
+          if (isDtFreeTextDef(this._currentDef)) {
```

Comment:
I am missing the check if we are on a 2nd level free-text. Can you point me in the right direction?

Reply:
Offline, I've discussed with Tom how the free-text editting is handled. Second layer and lower will reset the filter value upon edit to the first layer, resulting in the free-text not existing in the menu. This reason removes the necessity for second or lower layer support for free-text-edit-prefills